test/e2e/common/node/pods.go

```diff
@@ -638,7 +638,11 @@ var _ = SIGDescribe("Pods", func() {
 		})
 
 		ginkgo.By("submitting the pod to kubernetes")
-		podClient.CreateSync(ctx, pod)
+		pod = podClient.CreateSync(ctx, pod)
```

Comment:
I think we should modify `CreateSync` for wait for all containers running, so is what most of us assume,

Reply:
Yes, but not in this PR. Let's first see if it helps to get rid of flakes in this particular test. Agreed?